probe-rs/src/probe/cmsisdap/tools.rs

```diff
@@ -84,6 +84,9 @@ fn get_cmsisdap_info(device: &DeviceInfo) -> Option<DebugProbeInfo> {
                 interface.interface_number(),
                 interface_desc
             );
+            if prod_str.is_empty() {
+                prod_str = interface_desc;
```

Comment:
Please add a log entry here, to try and avoid confusion when an interface description ends up as the product name.

Reply:
which level should be used, trace or warn?